html/changelogs/DeadPoolJ.yml

```diff
@@ -0,0 +1,5 @@
+author: DeadPoolJ
+changes:
+	-rscadd: Adds the CELTIC core module and board, with the following law: "Be someone the crew can look up to and respect."
```

Comment:
Now how do I update the file without making a new PR?

This is the second PR I've done and I am new to Git.